File: test7.py
def flick_switch(lst):
    result, bul = [], True
    for i in lst:
        if i == 'flick':
            bul = not bul
        result.append(bul)

    return(result)


flick_switch(['bicycle', 'jarmony', 'flick', 'sheep', 'flick']);
flick_switch(['flick', 'chocolate', 'adventure', 'sunshine']);
flick_switch(['codewars', 'flick', 'code', 'wars']);

File: test7.js
function flickSwitch(arr){
    let flag = true
    for(let i = 0; i < arr.length; i++){
      if(arr[i] == 'flick'){
        flag == false ? flag = true : flag = false
      } 
      arr[i] = flag
    }
    return arr
  }

// Example usage
flickSwitch(['bicycle', 'jarmony', 'flick', 'sheep', 'flick']);
flickSwitch(['flick', 'chocolate', 'adventure', 'sunshine']);
flickSwitch(['codewars', 'flick', 'code', 'wars']);
